hooks/tk-multi-breakdown/basic/scene_operations.py

```diff
@@ -42,10 +42,15 @@ def scan_scene(self):
 
         refs = []
 
-        reference_template = self.parent.engine.get_template("reference_template")
+        # reference_template = self.parent.engine.get_template("reference_template")
```

Comment:
Should this line be removed?

Reply:
Thanks Stacey, I didn't pay attention to this line :)